<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eliminar paciente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 246, 255);
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Eliminar paciente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingrese la cedula del paciente a eliminar:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mensaje_e">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emensaje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>145</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ok1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cedulape">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="pixmap">
    <pixmap>lungs-2803208_960_720.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel2</sender>
   <signal>clicked()</signal>
   <receiver>cedulape</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
